docs/access-token.md

```diff
@@ -0,0 +1,92 @@
+# Access Token
+
+If you want to automate tasks with the Argo Server API or CLI, you will need an access token. 
+
+Firstly, create a role with minimal permissions. This example role for jenkins only permission to update and list workflows:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: jenkins-role
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - workflows
+    verbs:
+      - list
+      - update
+```
+
+Create a service account for your service:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: jenkins
+```
+
+Bind the service account to the role:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: jenkins
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: jenkins
+subjects:
+  - kind: ServiceAccount
+    name: jenkins
+```
+
+Create a secret:
```

Comment:
Not sure if this is intentional to create an extra secret for the SA. 

Maybe giving a way of using default SA secret, and also mentioning that extra secrets for same SA can be created (for different clients).

Reply:
I wanted to proved this as as simple way to change tokens. Do you have suggestion?

Reply:
Rotate the default token is even easier - just delete the secret. 

Rotating extra token requires `delete` and `create`.

Reply:
BTW - I don't have strong opinion on this, you decide it.

Reply:
I think you're idea is better. I've added it.